crates/core_simd/src/vector/float.rs

```diff
@@ -43,6 +43,15 @@ macro_rules! impl_float_vector {
             pub fn sqrt(self) -> Self {
                 unsafe { crate::intrinsics::simd_fsqrt(self) }
             }
+
+            /// Produces a vector where every lane has the reciprocal square root value
+            /// of the equivalently-indexed lane in `self`. Very useful for chaining multiplications
+            /// instead of divisions.
+            #[inline]
+            #[cfg(feature = "std")]
+            pub fn rsqrt(self) -> Self {
+                unsafe { _mm_rsqrt_ps(self) } // I'm missing the _mm256_rsqrt_ps version
```

Comment:
I think this requires more discussion. It's also not portable

Reply:
Yeah, we want this op, but that's not the way to do it.

Reply:
Also, `_mm_rsqrt_ps` is a very inaccurate approximation to reciprocal sqrt (iirc just 12 mantissa bits of the result are correct), for nearly all purposes you'd need something other than just that.

Reply:
Right, that's why I said it needs more discussion. To even use that intrinsic would require some decision on ffast-math stuff, and whatnot.